templates/boilerplate/config/blockchain.js

```diff
@@ -40,15 +40,28 @@ module.exports = {
     networkType: "custom",
     networkId: "1337",
     isDev: false,
-    genesisBlock: "config/privatenet/genesis.json", // Genesis block to initiate on first creation of a development node
     datadir: ".embark/privatenet/datadir",
-    mineWhenNeeded: true,
+    // -- mineWhenNeeded --
+    // This options is only valid when isDev is false. 
+    // Enabling this option uses our custom script to mine only when needed.
+    // Embark creates a development account for you (using `geth account new`) and funds the account. This account can be used for
+    // development (and even imported in to MetaMask). To enable correct usage, a password for this account must be specified
+    // in the `account > password` setting below.
+    // NOTE: once `mineWhenNeeded` is enabled, you must run an `embark reset` on your dApp before running
+    // `embark blockchain` or `embark run` for the first time.
+    mineWhenNeeded: true, 
+    // -- genesisBlock --
+    // This option is only valid when mineWhenNeeded is true (which is only valid if isDev is false).
+    // When enabled, geth uses POW to mine transactions at it would normally, instead of using POA as it does in --dev mode.
```

Comment:
maybe a typo here: at it would => as it would

Reply:
Good catch, thank you!